SS14.Admin/wwwroot/js/log-type-autocomplete.js

```diff
@@ -0,0 +1,223 @@
+﻿document.addEventListener('DOMContentLoaded', function () {
+    const LogType = {
```

Comment:
This list is quickly going to get out of date. It'd be best if SS14.Admin generates this list from the raw enum definition.